tests/panels/charts/datatable/test_datatable.py

```
@@ -0,0 +1,249 @@
+from dashboard_compiler.panels.charts.datatable.compile import (
+    compile_esql_datatable_chart,
+    compile_lens_datatable_chart,
+)
+from dashboard_compiler.panels.charts.datatable.config import ESQLDatatableChart, LensDatatableChart
+
+
+def compile_datatable_chart_snapshot(config: dict[str, Any], chart_type: str = 'lens') -> dict[str, Any]:
+    """Compile datatable chart config and return dict for snapshot testing."""
+    if chart_type == 'lens':
+        lens_chart = LensDatatableChart.model_validate(config)
+        layer_id, kbn_columns_by_id, kbn_state_visualization = compile_lens_datatable_chart(lens_datatable_chart=lens_chart)
```

Ruff (0.14.10)

Unpacked variable `kbn_columns_by_id` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)